Mathlib/NumberTheory/NumberField/Discriminant/Different.lean

```diff
@@ -20,12 +20,14 @@ import Mathlib.Tactic.Qify
 -/
 
 variable {K 𝒪 : Type*} [Field K] [NumberField K] [CommRing 𝒪] [Algebra 𝒪 K]
-variable [IsFractionRing 𝒪 K] [IsIntegralClosure 𝒪 ℤ K] [IsDedekindDomain 𝒪] [CharZero 𝒪]
-variable [Module.Finite ℤ 𝒪]
+variable [IsIntegralClosure 𝒪 ℤ K]
 
 open nonZeroDivisors
 
-lemma NumberField.absNorm_differentIdeal : (differentIdeal ℤ 𝒪).absNorm = (discr K).natAbs := by
+lemma NumberField.absNorm_differentIdeal [Module.Free ℤ 𝒪] [IsDedekindDomain 𝒪] :
```

Comment:
Don't you think it would be convenient to make `K` explicit here? To tell you the truth, I used this result in some future PRs and it is one of the first thing I did ;)